src/services/LocalizationService.ts

```diff
@@ -31,39 +28,366 @@ class LocalizationService
+                        nameFilter: "loremipsum64",
+                        yearFilter: "loremipsum65",
+                        semesterFilter: "loremipsum66",
+                        year: "loremipsum67",
+                        semester: "loremipsum68",
+                        telegramGroup: "loremipsum69",
+                        website: "loremipsum70",
+                        notices: "loremipsum71"
+                    },
+                    services: {
+                        text1: "loremipsum72",
+                        text2: "loremipsum73",
+                        availableServices: "loremipsum74",
+                        card1: { header: "loremipsum75", text: "loremipsum76" },
+                        card2: { header: "loremipsum77", text: "loremipsum78" },
+                        card3: { header: "loremipsum79", text: "loremipsum80" },
+                        card4: { header: "loremipsum81", text: "loremipsum82" },
+                        card5: { header: "loremipsum83", text: "loremipsum84" },
+                        card6: { header: "loremipsum85", text: "loremipsum86" },
+                        card7: { header: "loremipsum87", text: "loremipsum88" },
+                        card8: { header: "loremipsum89", text: "loremipsum90" },
+                        card9: { header: "loremipsum91", text: "loremipsum92" },
+                    },
+                    extraGroups: {
+                        text1: "loremipsum93",
+                        text2: "loremipsum94",
+                        availableGroups: "loremipsum95"
+                    },
+                    representatives: {
+                        text1: "loremipsum96",
+                        text2: "loremipsum97",
+                        departmentSelect: "loremipsum98"
+                    },
+                    contributors: {
+                        text1:"loremipsum99",
+                        header1: "loremipsum100",
+                        header2: "loremipsum101",
+                        dev1: "loremipsum102",
+                        dev2: "loremipsum103",
+                        dev3: "loremipsum104",
+                        githubProfile: "loremipsum105",
+                        websiteProfile: "loremipsum106",
+                        text2: "loremipsum107"
+                    },
                 },
 
-
-
-
                 en: {
-
-
-
-
+                
                     headerMenuItems: {
                         home: "Home",
                         aboutUs: "About Us",
                         rules: "Rules",
                         courses: "Courses",
-                        services: "Services",
-                        additionalGroups: "Extra groups",
-                        rapresentatives: "Rapresentatives",
-                        contributors: "Contributors"
+                        services: "All UniMi Services",
+                        additionalGroups: "Off-Topic Groups",
```

Comment:
This should remain "Additional Groups" or "Extra Groups".
All Unimi Services -> Unimi Services